python/cuspatial/cuspatial/_lib/interpolate.pyx

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
```

Comment:
We recently added these into `setup.py` in cudf, may want to copy that, but feel free to defer to a new PR.

Reply:
I will! :)